### ISSUE TITLE
Add CI for testing/wheels with Windows/Mac

### ISSUE THREAD

**Issue by menshikh-iv** [RaRe-Technologies/bounter#20]
This lib contains a lot of C code, for this reason, we need to make wheels for our users without compiler

- [ ] Setup Travis/CircleCI for Mac
- [ ] Setup AppVeyor for Windows
- [ ] [optionaly] Setup building wheels for Linux (with Travis)

**Comment by yaskevich** [comment]
I would like just to confirm it would be perfect to have wheels package for Windows.

**Comment by piskvorky** [comment]
Yeah, wheels would be nice. @yaskevich can you help out? (CC @mpenkov)

**Comment by yaskevich** [comment]
@piskvorky I am interested in this task to be completed. I never had experience with building wheels, but I have nothing against it. It there is a someone who could I ask about the details, it would be supercool. If not, I can try to do it by docs.

**Comment by piskvorky** [comment]
Building wheels is not difficult; the 3rd party tools and services (AppVeyor & co) will be the biggest pain. Hopefully @mpenkov and @menshikh-iv could answer your questions.

**Comment by yaskevich** [comment]
For starter, I built manually wheels package for Windows x64 and actual version of Python (3.7): https://github.com/yaskevich/bin/raw/master/bounter-1.1.0-cp37-cp37m-win_amd64.whl
After that, I played with AppVeyor. I connected it to my fork of bounter and managed to make AppVeyor work. It produced wheels package: https://ci.appveyor.com/api/buildjobs/vx0kub8yyax1rfff/artifacts/dist%2Fbounter-1.1.0-cp37-cp37m-win_amd64.whl
Binary mine and AppVeyor's are not the same, but I guess it's ok.
Now original bounter repo is to be connected to AppVeyor, as well uploading built packages to PyPI has to be set up (credentials will be required). Also there is a question for which architectures wheels packages should be produced. I set up only Python 3.7 for Windows x64. It seems I cannot connect AppVeyor to others repo.
By the way, is there any chat for bounter, gensim and stuff? Slack, Telegram, etc? I'm asking because I currently develop extension for gensim, and it would be cool to discuss some things.

**Comment by piskvorky** [comment]
There's [gitter](https://gitter.im/RaRe-Technologies/gensim) and there's [twitter](https://twitter.com/gensim_py). But the Gensim [mailing list](https://groups.google.com/forum/#!forum/gensim) and Github here are the liveliest :) What kind of extension are you working on?

@mpenkov can you please assist @yaskevich with setting up the release process for Bounter? Cheers.

**Comment by mpenkov** [comment]
@yaskevich Can you make a PR with the relevant AppVeyor config?

**Comment by yaskevich** [comment]
> There's [gitter](https://gitter.im/RaRe-Technologies/gensim) and there's [twitter](https://twitter.com/gensim_py). But the Gensim [mailing list](https://groups.google.com/forum/#!forum/gensim) and Github here are the liveliest :) What kind of extension are you working on?

Thanks for links. It's just a graph visualization builder, something like [this](https://vector.philology.by/). The question is whether it should be either a function of gensim, or standalone module.

> @mpenkov can you please assist @yaskevich with setting up the release process for Bounter? Cheers.
> @yaskevich Can you make a PR with the relevant AppVeyor config?

I've just made a PR #39. It is super simple, I guess it have to be more elaborate (e.g. support for more architectures, upload to PyPI).
